sonic-slave-bullseye/Dockerfile.j2

```diff
@@ -349,19 +353,24 @@ RUN apt-get update && apt-get install -y \
 
 RUN apt-get -y build-dep openssh
 
+# Build fix for ARM64 and ARMHF /etc/debian_version
+{%- if CONFIGURED_ARCH == "armhf" or CONFIGURED_ARCH == "arm64" %}
+RUN apt upgrade -y base-files
+{%- endif %}
```

Comment:
is this really an issue?

Reply:
Yes. This Bullseye arm64 container was last published well before the release of Bullseye. Because of that, `/etc/debian_version` contains `bullseye/sid` instead of 10.11, and after `sources.list` is changed to use our repo list, `lsb_release -cs` fails because that file doesn't contain numbers (details on that are [here](https://github.com/saiarcot895/sonic-buildimage/pull/7#discussion_r732025357)). Additionally, the container update for bullseye arm64 seems to have been [disabled in the multiarch repo](https://github.com/multiarch/debian-debootstrap/blob/master/.github/workflows/actions.yml#L27), although I'm not entirely sure why.

Updating the `base-files` package updates `/etc/debian_version` to the current version of the file, and then `lsb_release -cs` works normally.